src/main/resources/fabric.mod.json

```diff
@@ -22,7 +23,9 @@
     "disableportals.mixins.json"
   ],
   "depends": {
-    "fabricloader": ">=0.11.2",
-    "minecraft": "1.16.5"
+    "fabricloader": ">=0.14.6",
+    "fabric": "*",
+    "minecraft": "~1.19",
+    "java": ">=17"
```

Comment:
Why did you add a dependency on Fabric API?

Reply:
The mod's read me itself declared a dependency on the api - I made it official